app/assess/forms/comments_form.py

```diff
@@ -1,8 +1,9 @@
 from flask_wtf import FlaskForm
 from wtforms import TextAreaField
 from wtforms.validators import length
+from wtforms.validators import DataRequired
 
 
 class CommentsForm(FlaskForm):
 
-    comment = TextAreaField("Comment", validators=[length(max=10000)])
+    comment = TextAreaField("Comment", validators=[length(max=10000), DataRequired()])
```

Comment:
If you want a more friendly message, you can pass a `message` argument to the validator.